snap/snapcraft.yaml

```diff
@@ -1,39 +1,37 @@
 name: git-machete
-adopt-info: git-machete  # just for adopting `version`
 summary: git-machete is a versatile tool for organizing your git repo
 description: |
   git machete is a versatile tool for organizing your git repo, including features like:
   - Neat, customizable `git machete status` that shows what branches are in sync with their parent branch/remote tracking branch and which of them need to be rebased/merged/pulled/pushed
   - Semi-automatic `git machete traverse` through the branches that helps you effortlessly rebase/merge and push/pull the branches you care for
   - Automatic discovery of branch relations (`git machete discover`)
+adopt-info: git-machete # just for adopting `version`
```

Comment:
Any specific reason for these reorderings?

Reply:
@PawelLipski No, i initially did not know that there is snapcraft.yaml file already in the repo so i created mine from the scratch and only later found out the old one, i can reorder it so it matches previous version.